client/src/pages/Contest/Contest.tsx

```diff
@@ -0,0 +1,202 @@
+import { useState, useEffect } from 'react';
+import { useAuth } from '../../context/useAuthContext';
+import useStyles from './useStyles';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import Typography from '@material-ui/core/Typography';
+import Avatar from '@material-ui/core/Avatar';
+import Container from '@material-ui/core/Container';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Toolbar from '@material-ui/core/Toolbar';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import SubmittedDesigns from '../../components/SubmittedDesigns/SubmittedDesigns';
+import AuthHeader from '../../components/AuthHeader/AuthHeader';
+import ProfilePic from '../../Images/profilePic.png';
+import { Submission } from '../../interface/User';
+import { getContestById } from '../../helpers/APICalls/contest';
+
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
+import { Link } from 'react-router-dom';
+
+import { withStyles } from '@material-ui/core/styles';
+
+const GlobalCss = withStyles({
+  // @global is handled by jss-plugin-global.
+  '@global': {
+    // You should target [class*="MuiButton-root"] instead if you nest themes.
+    '.MuiToolbar-root': {
+      padding: 0,
+    },
+    '.MuiToolbar-regular': {
+      minHeight: 0,
+    },
+    'li.MuiImageListItem-root': {
+      height: '250px !important',
+      overflowY: 'hidden',
+    },
+    '.MuiImageListItemBar-root': {
+      background: 'none',
+    },
+    'ul.li.MuiImageListItem-item': {
+      padding: 0,
+      height: '250px !important',
+    },
+    '.MuiImageListItemBar-titleWrap': {
+      color: '#333',
+      marginTop: '12px',
+    },
+    '.MuiImageListItemBar-title': {
+      fontSize: '15px',
+      fontWeight: 'bolder',
+      textDecoration: 'underline',
+    },
+    '.MuiImageListItem-imgFullHeight': {
+      opacity: '0.8',
+    },
+  },
+})(() => null);
+
+export default function Contest(): JSX.Element {
+  const classes = useStyles();
+  const [value, setValue] = useState(0);
+  //const [contestSubmissions, setContestSubmissions] = useState<Submission[]>([]);
+  const [contestSubmissions, setContestSubmissions] = useState<any>([]);
```

Comment:
Try to avoid using `any` as the type

Reply:
currently the api is not integrated, so temporarily i am pushing the code using any, but i will make changes in the future after fetching from the api is fully working. I could not find any other ways to avoid it for now.